### ISSUE TITLE
Examples don't work

### ISSUE THREAD

**Issue by RowanDZ** [mdn/content#33636]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/CustomStateSet

### What specific section or headline is this issue about?

Examples

### What information was incorrect, unhelpful, or incomplete?

Examples do not work

### What did you expect to see?

Working examples

### Do you have anything more you want to share?

They give:
```
SyntaxError: Failed to execute 'add' on 'CustomStateSet': The specified value 'checked' must start with '--'.
```

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/customstateset`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/CustomStateSet
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/customstateset/index.md
* Last commit: https://github.com/mdn/content/commit/7a903274c7a99096c3a2c82f6924af3e6b9a35a7
* Document last modified: 2024-04-16T01:03:53.000Z

</details>

**Comment by minimalsm** [comment]
Hey folks :) happy to put up a fix for this.

**Comment by hamishwillee** [comment]
Thanks, but the examples do work if you try them on a browser that supports the latest syntax, such as Firefox. This is marked in the page:

![image](https://github.com/mdn/content/assets/5368500/7a36fa2e-bf4d-4eee-8035-974989e2468b)

This will fix itself a bit in the next few weeks as Chrome also supports the new CSS `:state()` syntax from Chrome 125 - https://github.com/mdn/browser-compat-data/pull/23072

I appreciate this is a bit confusing based on the compatibility data. I've updated the page a bit in https://github.com/mdn/content/pull/33572 too.

In any case, I propose we do not modify the examples to support the syntax that is no longer allowed by the spec.

Closing this, but if you wanted we'd accept a PR that modifies the examples to check for support of the new state syntax and put a "not supported by this browser warning" when the examples don't work.
